typer.js

```diff
@@ -5,9 +5,10 @@ var Typer = function(element) {
   var words = element.dataset.words || "override these,sample typing";
   this.words = words.split(delim).filter(function(v){return v;}); // non empty words
   this.delay = element.dataset.delay || 200;
+  this.loop = element.dataset.loop || true;
```

Comment:
You updated to string "false" on line 65, and I think this one should be updated as well to ... || "true".